src/components/AccountModule/connection-hooks.js

```diff
@@ -214,6 +221,18 @@ export function useSyncState(
   const currentBlock = getBlockNumber()
 
   const minimumTransactionBalance = new BN(0.005)
+  const networkName = network.shortName.toLowerCase()
+
+  if (
+    clientListening &&
+    (networkName === 'local' || networkName === 'unknown')
+  ) {
+    return {
+      header: 'Synced',
+      info: currentBlock ? `: current block ${currentBlock}` : '',
+      status: STATUS_CONNECTION_OK,
+    }
```

Comment:
Similarly, maybe we should provide a single const variable for this default working status